app/util/appUtil.js

```diff
@@ -125,6 +125,44 @@ async function getLastTokenId() {
   return lastTokenId ? parseInt(lastTokenId, 10) : 0
 }
 
+function membershipValidator(members) {
+  const keyring = new Keyring({ type: 'sr25519' })
+  const alice = keyring.addFromUri(USER_URI)
+  const validMembers = [alice.address]
+
+  const validatedMembers = validMembers.reduce((acc, item) => {
+    if (members.includes(item)) {
+      acc.push(item)
+      return acc
+    }
+  }, [])
+
+  return { validMembers, validatedMembers, validMembersResult: validMembers.length === validatedMembers.length }
+}
```

Comment:
I'm not sure what this is trying to achieve. Alice won't always be present in all environments so the question is what is this check trying to represent?

Reply:
Validating that a member (although hardcoded) exists within the membership list.

I had a question around this, validating via passing a name in for example doesn't sound feasible but would that suffice?